__tests__/github_service_test.ts

```diff
@@ -115,4 +116,49 @@ describe("Github Tests" , () => {
       })
   });
 
+  it('should return exposed container port', (done: any) => {
+    const gr = new GitSource(
+      "https://github.com/mikesparr/tutorial-react-docker",
+      SecretType.NO_AUTH,
+      null
+    );
+
+    const gs = new GithubService(gr);
+    gs.getDockerfileContent()
+      .then((content: string) => {
+        const parser = new DockerFileParser(content);
+        const port = parser.getContainerPort();
+        expect(port).toEqual(5000);
+        done();
+      })
+      .catch((err: Error) => done(err))
+  });
+
+  it('should detect Dockerfile', (done: any) => {
```

Comment:
I see there is a test `should not return exposed container port` for BitBucket. A similar test for Github is missing.